bokeh/models/widgets/inputs.py

```diff
@@ -152,6 +152,10 @@ class Slider(InputWidget):
     A callback to run in the browser whenever the current Slider value changes.
     """)
 
+    delay_callback = Bool(default = False, help="""
```

Comment:
Corrected.   (Not sure why the spaces cause issues though.  I think they help for readability.)